src/utils/clients/discogs/formatters/album-search-response.formatter.ts

```diff
@@ -0,0 +1,31 @@
+interface AlbumResult {
+  type: string;
+  id: number;
+  master_id?: number;
+  title: string;
+  year: number;
+  thumb: string;
+  cover_image: string;
+}
```

Comment:
I don't see the point of having an interface for the incoming (unformatted) data, as we can't control what comes from Discogs API and it will be a burden for us maintaining it. I'd rather take the input as an `any` type and define an interface for our album objects (i.e. after the transform).

Reply:
Wanted to avoid the any type and didn't hurt to add the interface here.

Reply:
Changed to any in 8a2863f